cms/static/js/views/settings/main.js

```diff
@@ -403,8 +390,15 @@ define(['js/views/validation', 'codemirror', 'underscore', 'jquery', 'jquery.ui'
           extended_valid_elements: "*[*]",
           invalid_elements: "",
           browser_spellcheck: true,
-          content_css: window.fecUrl('stylesheets/tinymce.min.css')
-        });
+        };
+
+        TinyMCE.baseURL = baseUrl + "/js/vendor/tinymce/js/tinymce";
+        TinyMCE.init(Object.assign(tinymceOptions, window.CustomizeFunctionsHook.getTinymceExtraOptions({
```

Comment:
Hi @tranphan-groove ,

Check window.CustomizeFunctionsHook and window.CustomizeFunctionsHook.getTinymceExtraOptions before using it